tests/queries/0_stateless/02956_fix_to_start_of_milli_microsecond.sql

```diff
@@ -0,0 +1,2 @@
+SELECT toStartOfInterval(toDateTime64('2023-10-09 10:11:12.000999', 6), toIntervalMillisecond(1));
```

Comment:
It would be also interesting to check the behaviour on `.000500` and `.000499` subseconds